src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -1,645 +1,11 @@
-                {
-                    Log.LogMessage($"Specified directory, {directory}, does not exist");
-                }
-                else
-                {
-                    GiveFullPermissionsIfUnixFolder(directory);
-
-                    Log.LogMessage($"Attempting to cleanup {directory} ... ");
-#if net45
-                    // Unlike OSX and Linux, Windows has a hard limit of 260 chars on paths.
-                    // Some build definitions leave paths this long behind.  It's unusual, 
-                    // but robocopy has been on Windows by default since XP and understands 
-                    // how to stomp on long paths, so we'll use it to clean directories on Windows first.
-                    if (EnableLongPathRemoval && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                    {
-                        RemoveDirectoryRecursiveLongPath(directory);
-                    }
-                    else
-                    {
-#endif
-                        Directory.Delete(directory, true);
-#if net45
-                    }
-#endif
-                    Log.LogMessage("Success");
-
-                    // For additional directories we want only the contents deleted but not the root. The easiest is just to recreate the root folder
-                    // after the whole folder has been created. Since this just applies to additional ('known') folders we should not worry about long paths.
-                    if (isAdditionalDirectory && !Directory.Exists(directory))
-                    {
-                        Directory.CreateDirectory(directory);
-                    }
-                }
-                return true;
-            }
-            catch (Exception e)
-            {
-                if (ignoreExceptions)
-                {
-                    Log.LogMessage($"Failed in cleanup attempt of '{directory}', but directory is non-blocking, ignoring failure.");
-                    Log.LogMessage($"{e.GetType().ToString()} - {e.Message}");
-                    return true;
-                }
-                else
-                {
-                    attempts++;
-                    Log.LogMessage($"Failed in cleanup attempt... {Retries - attempts} retries left.");
-                    Log.LogMessage($"{e.GetType().ToString()} - {e.Message}");
-                    Log.LogMessage(e.StackTrace);
-                    if (attempts < Retries)
-                    {
-                        Log.LogMessage($"Will retry again in {SleepTimeInMilliseconds} ms");
-                        await System.Threading.Tasks.Task.Delay(SleepTimeInMilliseconds.Value);
-                        return await CleanupDirectoryAsync(directory, attempts).ConfigureAwait(false);
-                    }
-                }
-            }
-            Log.LogMessage("Failed to cleanup.");
+            Log.LogMessage($"This BuildTask has been deprecated.");
             return false;
```

Comment:
There's no reason to fail over a deprecated task.  Also might mention why (in favor of maintenance jobs)

Reply:
Thanks. Fixed.